### PR TITLE
109 beta post

### PR DESCRIPTION
**Pull request by rachelandrew**
Adds the beta post.

Preview: https://deploy-preview-4448--developer-chrome-com.netlify.app/blog/chrome-109-beta/

### CODE REVIEW

**Comment by rachelandrew** [comment]
@AaronForinton could you give this a quick review, for any horrors I've missed while copy and pasting it all together :) It's waiting for the beta to actually go out before it is merged.

File: site/en/blog/chrome-109-beta/index.md
Comment (AaronForinton) [inline site/en/blog/chrome-109-beta/index.md:35]:
```
This can cause a visible 1px gap between a parent element's border and a child's background when the border is rounded down during paint, since the snapping floors the value but the layout rounded it up to calculate the child’s position (for example, border-width is set to 10.75 pixels, rounded to 10px at paint time, but 11 at layout time).
```

I'm not really clear on what this means. I think it could do with reworking into 2 sentences.

Comment (AaronForinton) [reply site/en/blog/chrome-109-beta/index.md:35]: @rachelandrew Just one comment. I committed a couple of tiny things

File: site/en/blog/chrome-109-beta/index.md
Comment (agektmr) [inline site/en/blog/chrome-109-beta/index.md:51]: Somehow this was dropped. We are publishing [a new blog post for this](https://github.com/GoogleChrome/developer.chrome.com/pull/4478).

```suggestion

### Secure Payment Confirmation on Android Chrome

[Chrome 109 on Android supports Secure Payment Confirmation (SPC)](/blog/spc-on-android), which is a [proposed web standard](https://www.w3.org/TR/secure-payment-confirmation/) that allows customers to authenticate with a credit card issuer, bank, or other payment service provider using a platform authenticator — typically activated with a device's screen unlock feature such as a fingerprint sensor. This usually happens during a payments authentication protocol such as [EMV 3-D Secure](https://www.emvco.com/emv-technologies/3d-secure/) or [Open Banking](https://standards.openbanking.org.uk/). EMV 3-D Secure, for example, has support for SPC in its [v2.3 spec release](https://www.emvco.com/emv_insights_post/what-is-new-with-emv-3ds-v2-3/). We [previously announced](/articles/secure-payment-confirmation/) that SPC was launched for Google Chrome on macOS and Windows and provided developer guides for both [registration](/articles/register-secure-payment-confirmation/) and [authentication](/articles/authenticate-secure-payment-confirmation/).

```

**Review by agektmr — CHANGES_REQUESTED** [review]

File: site/en/blog/chrome-109-beta/index.md
Comment (rachelandrew) [reply site/en/blog/chrome-109-beta/index.md:51]: @agektmr this isn't on the Chrome Status roadmap therefore isn't in the post. So it was't dropped, it wasn't in the data. If you want things to be included in the beta post, ensure they are showing up on the roadmap.

File: site/en/blog/chrome-109-beta/index.md
Comment (rachelandrew) [inline site/en/blog/chrome-109-beta/index.md:35]:
```suggestion
Currently Blink snaps the border widths at paint time. This can cause a visible 1px gap between a parent element's border and a child's background when the border is rounded down during paint. This happens because the snapping floors the value, but the layout rounds it up to calculate the child’s position (for example, border-width is set to 10.75 pixels, rounded to 10px at paint time, but 11 at layout time). By making this change Blink will behave like Gecko and WebKit, improving interoperability.
```

**Pull request merged**
